src/nnvm/node_op_util.h

```diff
@@ -80,6 +80,12 @@ class NodeOpGen {
                                                    dependent_node->attrs.name + "_negative",
                                                    {x}, nullptr, &dependent_node)};
     }
+
+    nnvm::NodeEntry ones_like(const nnvm::NodeEntry &x) {
```

Comment:
Don't we already have an ones_like operator already for use? https://github.com/apache/incubator-mxnet/blob/971ea4f0adb5bd52cdcdee3e15f390ac05a3d379/src/operator/tensor/init_op.cc#L185